pkg/controller/volume/attachdetach/populator/desired_state_of_world_populator.go

```diff
@@ -113,3 +126,21 @@ func (dswp *desiredStateOfWorldPopulator) findAndRemoveDeletedPods() {
 		dswp.desiredStateOfWorld.DeletePod(dswPodUID, dswPodToAdd.VolumeName, dswPodToAdd.NodeName)
 	}
 }
+
+func (dswp *desiredStateOfWorldPopulator) findAndAddActivePods() {
+	pods, err := dswp.podLister.List(labels.Everything())
```

Comment:
@gnufied that's a `PodLister` backed by the shared informer's cache. It's not hitting the apiserver. There is no data going over the wire for this call.

If you have a cluster with 100k or 250k or more pods, do you think it's safe to iterate through them every minute, even if it's just pulling them out of a cache?